elements/lisk-transactions/test/register_multisignature_account.spec.ts

```diff
@@ -88,80 +92,121 @@ describe('#registerMultisignature transaction', () => {
-		describe('when the register multisignature transaction is created without a passphrase', () => {
-			beforeEach(() => {});
-
-			describe('validation errors', () => {
-				describe('when lifetime', () => {
-					it('was not provided', () => {});
-
-					it('is float', () => {});
-
-					it('is not number type', () => {});
-
-					it('was more than expected', () => {});
-
-					it('was less than expected', () => {});
-				});
-			});
+		it('should throw when fee is invalid', async () => {
+			(input as any).fee = 'invalid_shold_be_number_string';
+			expect(() => registerMultisignature(input)).toThrow(
+				'Fee must be a valid number in string format.',
+			);
+		});
 
-			describe('when minimum', () => {
-				it('was not provided', () => {});
+		it('should throw when number of signatures is less than lower limit', async () => {
+			(input as any).numberOfSignatures = 0;
+			expect(() => registerMultisignature(input)).toThrow(
+				'Please provide a valid numberOfSignatures value. Expected integer between 1 and 64.',
+			);
+		});
 
-				it('is float', () => {});
+		it('should throw when number of signatures is more than higher limit', async () => {
+			(input as any).numberOfSignatures = 65;
+			expect(() => registerMultisignature(input)).toThrow(
+				'Please provide a valid numberOfSignatures value. Expected integer between 1 and 64.',
+			);
+		});
 
-				it('is not number type', () => {});
+		it('should throw when number of signatures less than mandatory keys', async () => {
+			(input as any).mandatoryKeys.push('fffffffffffffffffffff');
+			(input as any).numberOfSignatures = 2;
+			expect(() => registerMultisignature(input)).toThrow(
+				'The numberOfSignatures should be more than or equal to the number of mandatory passphrases.',
+			);
+		});
 
-				it('was more than expected', () => {});
+		it('should throw when number of signatures is bigger than the count of optional and mandatory keys', async () => {
+			(input as any).numberOfSignatures = 5;
+			expect(() => registerMultisignature(input)).toThrow(
+				'Please provide a valid numberOfSignatures. numberOfSignatures (5) is bigger than the count of optional (2) and mandatory (2) keys.',
+			);
+		});
 
-				it('was less than expected', () => {});
-			});
+		it('should throw error if Network Identifier is empty', async () => {
+			(input as any).networkIdentifier = '';
+			expect(() => registerMultisignature(input)).toThrow(
+				'Network identifier can not be empty.',
+			);
+		});
 
-			it('should have the type', () => {});
+		it('should throw error if duplicate keys are found', async () => {
+			(input as any).optionalKeys[0] = (input as any).mandatoryKeys[0];
+			expect(() => registerMultisignature(input)).toThrow(
+				`There are repeated values in optional and mandatory keys: '4a67646a446313db964c39370359845c52fce9225a3929770ef41448c258fd39'`,
+			);
+		});
 
-			it('should have the fee', () => {});
+		it('should throw error if repeated mandatory keys exists', async () => {
```

Comment:
I see `repeated` and `duplicate` phrase used differently, maybe everywhere we can use `duplicate`?

Reply:
could be repeated more than once so it's different :)